tethys_portal/consumers/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""
+********************************************************************************
+* Name: consumers/__init__.py
+* Author: Michael Souffront
+* Created On: 2019
+* Copyright: (c) Aquaveo, LLC 2019
```

Comment:
@sdc50 @SarvaPulla I'm not a legal expert, but what should the copyrights be in our Tethys Platform headers? I realized many of my contributions have Aquaveo listed as the copyright holder as well, b/c my editor adds it automatically.

Reply:
That's probably a good discussion for scrum/steering committee meeting. I believe Dan Ames might be able to help guide the discussion, since he has some experience in open source copyrights.

Reply:
I haven't read this yet, but this series seems like it would be helpful: https://haacked.com/archive/2006/01/24/DevelopersGuideToOpenSourceSoftwareLicensing.aspx/